iotilecore/iotile/core/hw/hwmanager.py

```diff
@@ -34,7 +39,7 @@
 
 
 @context("HardwareManager")
-class HardwareManager(object):
+class HardwareManager:
```

Comment:
Quick question.  My understanding was that in python 2.7, you still needed to add the `(object)` explicitly to make this a new-style class and that was only unnecessary on python 3+?  Am I missing something?

Reply:
I may be misreading, but in python's docs they claim new-style classes have been integrated with 2.7 https://www.python.org/doc/newstyle/

Reply:
Yeah it looks like you're right, til:

```
Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> class test_class: pass
...
>>> obj = test_class()
>>> isinstance(obj, object)
True
```

Reply:
It's confusing because they say they haven't updated the python docs, I think it's probably safer to keep (object) in place. I'll add them in, more important than fixing a python3 linting error.